### PR TITLE
detect/lua/v1: add tests for detect-lua

### PR DESCRIPTION
**Pull request by TheKharleeci**
Includes test for LuaMatchTest01

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

### CODE REVIEW

File: tests/detect-lua-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
+- filter:
+    count: 2
+    match:
+      event_type: alert
+      alert.signature_id: 6677001
```
Comment (jufajardini) [inline tests/detect-lua-01/test.yaml:21]: Considering there are two other rules added to the test, can you add filters for those as well, or, if they're not needed for the test, considering the original ones in detect-lua, you could probably remove them.

**Review by jufajardini — CHANGES_REQUESTED** [review]
Please check inline comment :)

The CI failure should be fixed by adding a suricata.yaml file, to prevent S-V to looking at the default location for the lua script, I believe.

File: tests/detect-lua-01/test.yaml (hunk 1 above)
Comment (catenacyber) [reply tests/detect-lua-01/test.yaml:21]: friendly ping @TheKharleeci

Comment (TheKharleeci) [reply tests/detect-lua-01/test.yaml:21]: Thanks for the friendly ping @catenacyber. I think I'm confused on how to go about this particular test

Comment (catenacyber) [reply tests/detect-lua-01/test.yaml:21]: How can we help ?
What about the comments from Juliana ?

Comment (TheKharleeci) [reply tests/detect-lua-01/test.yaml:21]: Currently working on it. Apologies for the late reply

**Comment by catenacyber** [comment]
Is this still live ?

**Comment by TheKharleeci** [comment]
> Is this still live ?

yes please

**Comment by TheKharleeci** [comment]
Followed By: #1086

**Pull request closed without merging**